include/OpenSpaceToolkit/Astrodynamics/EventCondition.hpp

```diff
@@ -28,16 +29,20 @@ class EventCondition
     /// @brief                  Constructor
     ///
     /// @code
-    ///                         EventCondition eventCondition = {aName, aCriterion, anEvaluatro, aTarget};
+    ///                         EventCondition eventCondition = {aName, anEvaluator, aTarget, targetIsRelative};
     /// @endcode
     ///
     /// @param                  [in] aName A string representing the name of the Real Event Condition
-    /// @param                  [in] aCriterion An enum indicating the criterion used to determine if the Real Event
-    /// Condition is met
-    /// @param                  [in] anEvaluator A function evaluating a state and a time
+    /// @param                  [in] anEvaluator A function evaluating a state
     /// @param                  [in] aTarget A target value associated with the Real Event Condition
+    /// @param                  [in] targetIsRelative A boolean indicating whether or not the target is relative
 
-    EventCondition(const String& aName);
+    EventCondition(
```

Comment:
Is this an interface breaking change? FWIW I still like the idea semantically of just making Target a monad (i.e. either real valued or relative valued) but agree not worth a major breaking change just for that 😛 

One (perhaps big) plus, though, is that we wouldn't have to propagate down an extra argument to all of the event condition implementation children.

Reply:
This is an interface breaking change for the EventCondition class, but none of the children, so technically things should still work, since the EventCondition is just an abstract class.
I do like the Monad idea I might play with that in this MR on another branch.